tests/components/freebox/test_utils.py

```diff
@@ -0,0 +1,21 @@
+"""Tests for the Freebox utility methods."""
+import json
+
+from homeassistant.components.freebox.router import is_json
+
+from .const import DATA_LAN_GET_HOSTS_LIST_MODE_BRIDGE, WIFI_GET_GLOBAL_CONFIG
+
+
+async def test_is_json() -> None:
```

Comment:
This code is in `router.py`, so it `test_router.py`

Reply:
Done